### ISSUE TITLE
Vanilla Snapshot 22w43a will not download/launch

### ISSUE THREAD

**Issue by troywweber7** [killpowa/Twickt-Launcher#1467]
- [X] I searched the issue tracker for other issues covering my case
- [X] This is a Launcher issue. (Usually if you can reproduce it with a fresh instance)

**Describe the bug**
Cannot download 22w43a.

**To Reproduce**

1. Open GDLauncher 1.1.29
2. Create a new instance for Vanilla Snapshot 22w43a and wait for it to download game files
3. Launcher will download twice and there will be an error message in the console

**Expected behavior**
Downloads & launches 22w43a without errors.

**Possible solutions**
I could download the vanilla launcher and launch through there, I suppose.

**Screenshots**
This is the version I tried to create/update.
![image](https://user-images.githubusercontent.com/4765979/198144911-f9f430ba-93a9-4bf5-becb-eff443130781.png)

When trying to download or when you click repair and it tries to download again.
![image](https://user-images.githubusercontent.com/4765979/198144977-14f78b41-1c2c-4d08-ac6e-4315cb4414aa.png)

Error in the console after trying to download.
![image](https://user-images.githubusercontent.com/4765979/198145028-0fb396b0-325b-4949-ba30-d0216f9b0055.png)

**Operating System:**
- OS: Ubuntu 20.04
- Launcher Version: 1.1.29
- Java Version: 8/17
- Minecraft Version: 22w42a
- Mod Loader: NA, Vanilla

**Additional context**
No additional context. Thanks for your help!

**Comment by Eskaan** [comment]
Duplicate of #1463
Yes, it is another snapshot, but it is extremely likely that it's the same cause.
Edit: My fault, this is a separate issue.

**Comment by troywweber7** [comment]
@Eskaan the console logs are different, but I guess if you are going to work on them at the same time then they kind of are duplicate.

**Comment by Eskaan** [comment]
They are just taken at different times I think. I assume your was taken after downloading and he closed the launcher in between.

**Comment by troywweber7** [comment]
@Eskaan that's incorrect, actually. I've submitted both of these issues and the screenshots are not taken at different times. These are two different errors for two different snapshots (22w42a vs. 22w43a).

I did not provide screenshots in the previous issue because the screenshots the other individual supplied exactly mirrored my case and I felt it irrelevant to add duplicate screenshots. Perhaps I should have verified on that issue that his screenshots were identical to mine, however, I apologize for the confusion.

As such my advice is to not mark this as a duplicate.

**Comment by Eskaan** [comment]
Will check the issue myself later today, won't close this issue until I confirmed what I think.
